### ISSUE TITLE
Linux Mint Xfce 17.1: redshift-gtk does not show in panel

### ISSUE THREAD

**Issue by gordon-morehouse** [jonls/redshift#219]
32-bit on a Core 2 Duo. redshift-gtk, when autostarted or started manually, may not show in the panel (tasktray). redshift and gtk-redshift 1.9.1.1-trusty.

**Comment by maandree** [comment]
Do you have all dependencies installed?
python3, pygobject, pyxdg, appindicator

**Comment by gordon-morehouse** [comment]
I'll check when I'm at the machine; if they are not installed, would this be an Ubuntu and/or Mint package dependency bug? I installed redshift-gtk from 'mintinstall'.

**Comment by maandree** [comment]
No, they are optional dependencies to get redshift-gtk working.

**Comment by TingPing** [comment]
> No, they are optional dependencies to get redshift-gtk working.

They should still be "Recommended" by the package and installed by default.
